### PR TITLE
3468 dispute page responsive layout

### PR DESCRIPTION
**Pull request by bconfortin**
I think the only part missing is the 3 column layout for tablets.

https://github.com/AugurProject/augur/issues/3468

### CODE REVIEW

File: packages/augur-ui/src/modules/market-cards/common.tsx [hunk 1]
```diff
@@ -88,22 +88,24 @@ export const DisputeOutcome = (props: DisputeOutcomeProps) => {
...
-              ? calculatePosition(
-                  ZERO,
-                  createBigNumber(bondSizeCurrent.value),
-                  stakeCurrent
-                )
-              : 0
-          }
-        />
-      )}
+      <div>
```
Comment (bthaile) [inline packages/augur-ui/src/modules/market-cards/common.tsx]: shouldn't add <div> tags just for styling. CSS should handle it

File: packages/augur-ui/src/modules/market-cards/market-card.styles.less [hunk 2]
```diff
@@ -277,6 +277,64 @@
...
+				margin: 0;
+			}
+
+			> span:nth-of-type(3) {
+				display: none;
+			}
 		}
 	}
+
+	// @media @breakpoint-mobile-new {
```
Comment (bthaile) [inline packages/augur-ui/src/modules/market-cards/market-card.styles.less]: don't leave commented out CSS code.

File: packages/augur-ui/src/modules/portfolio/components/common/box-header.styles.less [hunk 3]
```diff
@@ -75,9 +89,18 @@
...
+.DisputeTabs {
+  background: @color-table-header;
+  border: 1px solid @color-dark-grey;
   margin-top: @size-16;
+
+  @media @breakpoint-tablet {
+    margin-top: 0;
+  }
+
+  @media @breakpoint-mobile-new {
```
Comment (bthaile) [inline packages/augur-ui/src/modules/portfolio/components/common/box-header.styles.less]: not sure what @breakpoint-mobile-new means why add `new` isn't descriptive

File: packages/augur-ui/src/modules/portfolio/components/common/box-header.tsx [hunk 4]
```diff
@@ -27,22 +27,14 @@ export interface BoxHeaderProps {
...
-      </div>
       <div
         className={classNames(Styles.Middle, {
           [Styles.isSwitched]: props.switchHeaders,
         })}
       >
         {props.bottomBarContent && (
           <div
-            className={classNames(Styles.BottomContent, {
+            className={classNames(Styles.DisputeTabs, {
```
Comment (bthaile) [inline packages/augur-ui/src/modules/portfolio/components/common/box-header.tsx]: this is a common component shouldn't have any references to impl `DisputeTabs` naming doesn't belong here

File: packages/augur-ui/src/modules/reporting/common.styles.less [hunk 5]
```diff
@@ -246,6 +246,11 @@ div.ReportingSubheaders {
   @media @breakpoint-mobile {
     width: unset;
   }
+  
+  @media @breakpoint-mobile-new {
```
Comment (bthaile) [inline packages/augur-ui/src/modules/reporting/common.styles.less:270]: already mentioned, don't like the `@breakpoint-mobile-new` naming isn't descriptive enough

File: packages/augur-ui/src/modules/reporting/common.styles.less [hunk 6]
```diff
@@ -599,6 +604,11 @@ div.ReportingSubheaders {
   max-width: 482px;
   padding: @size-32 @size-24;
 
+  @media @breakpoint-mobile-new {
```
Comment (bthaile) [inline packages/augur-ui/src/modules/reporting/common.styles.less:627]: already mentioned, don't like the `@breakpoint-mobile-new` naming isn't descriptive enough

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 💙

 This link will always point to this PR's latest changes: https://peek.run/vv2hzq5

**Review by bthaile — COMMENTED** [review]
let me know when you back out changes you don't want and we'll get this merged in

**Comment by bconfortin** [comment]
> let me know when you back out changes you don't want and we'll get this merged in

@bthaile Guess that's done. Will do further testing tomorrow, it's already 11:18 PM here.

**Review by bthaile — APPROVED** [review]
lets get this in, looks great there are some edge cases where sizing is off. Could be from resizing the web browser. should look good on actual devices.

**Pull request merged**